packages/react-components/react-portal-compat-context/package.json

```diff
@@ -16,7 +16,7 @@
     "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
-    "lint": "just-scripts lint && just-scripts lint-imports",
+    "lint": "just-scripts lint",
```

Comment:
this package will run lint-imports narrowed to only AMD checks as a part of `build` task post-step. The need of this check is configured by `tags` within `workspace.json`